spec/post_index_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
```

Comment:
- [ ] **[OPTIONAL]** I think it would be best to move all these features you added into a new folder as you have done with the requests specs, so your codebase looks more tidy.